delete-unverified-users/src/main/java/eu/europeana/keycloak/usermgt/DeleteUnverifiedUserProvider.java

```diff
@@ -69,54 +69,92 @@ public Object getResource() {
     @GET
     @Produces({MediaType.APPLICATION_JSON})
     public String delete(
-            @DefaultValue("1") @QueryParam("age") int minimumAge) {
-//        return removeUnverifiedUsers(minimumAge);
+        @DefaultValue("1") @QueryParam("age") int minimumAgeInDays) {
+//        return removeUnverifiedUsers(minimumAgeInDays);
```

Comment:
Why is this commented out? Shouldn't this method really delete things, so do the call to removeUnverifiesUsers at the end of the method?